common/viewer.cpp

```diff
@@ -2493,6 +2681,11 @@ namespace rs2
         return false;
     }
 
+    std::shared_ptr<texture_buffer> viewer_model::get_last_texture()
```

Comment:
This seem to be a public member - what is the getter for ?

Reply:
last_texture is private